spec/requests/acuant_sdk_spec.rb

```diff
@@ -1,5 +1,19 @@
 require 'rails_helper'
 
+describe 'acuant sdk versions' do
+  default_version = IdentityConfig.store.idv_acuant_sdk_version_default
+  alternate_version = IdentityConfig.store.idv_acuant_sdk_version_alternate
+  acuant_entries = Dir.entries('public/acuant/')
```

Comment:
Wonder if we should use `Rails.root` here, since it may resolve relative to the current working directory (unsure what use-case someone would be running tests other than from root of project, but seems to be a best-practice, also see line 18 below).